### ISSUE TITLE
MILESTONE: Clear, Defined Goals

### ISSUE THREAD

**Issue by sevvie** [zlisters/wiki#5]
Seriously, what the fuck are we actually doing? There's like a thousand ideas from a thousand proud individualists -- no, fuck that. I'm taking control of this madcap anarchy and chaos. There are three general ideas around which we orbit seem to be: Expanded/Backup Hosting, Database of Research, and a Shared Network Resource System, not to mention the various tools we want to make. The most important part, right now, is producing Minimum Viable Product, or the barest-bones solutions that meet design expectations. We all have a lot of ideas, but we have to take the machete to them for now, and put the extended things off until we have something to which we can add it.

**Comment by sevvie** [comment]
Speaking of which, the other tech dorks on the team need github accounts and need to give me their account names.

**Comment by Tom-Sequitur** [comment]
"I'm taking control of this madcap anarchy and chaos."  Well I take exception to that.  You're welcome to start your own project, or even recruit from the work and volunteers that I have gathered.  But if we're talking about project leads being decided on by "I say so" -- well good luck.  Most of the people involved at this point have been informed of the plan to organize this thing and if we deviate from the plan that's not fair to those folks.

That being said that's why people need to come up with proposals and get them listed out.  And some of those ideas are posted on the discord server and some are here: http://greathall.org/documents/projectstatus.pdf

My thoughts are that this is what the meetings are for and we can take a month or so of meetings to sort of hash out what we think are the best proposals and hopefully settle on project leads for this or that. Like I said, I am going to put together the data schema that I think meets what I want to accomplish and I have already posted two videos on different parts of the site as proposals.

**Comment by Tom-Sequitur** [comment]
The dorks are shown in the discord membership and also in the collab doc (the ones that made it to today's meetings).

**Comment by sevvie** [comment]
> "I'm taking control of this madcap anarchy and chaos." Well I take exception to that. 

Forgive me, it was more of a half-trolling rousing speech than anything else. I don't plan on being a dictator, just... providing what I have to share toward the greater whole, which includes software project management.

**Comment by Tom-Sequitur** [comment]
Yeah I don't have a problem with folks putting their hat in the ring. And I am not qualified. I do want to see folks come to a consensus on project leads, but my vote is just one vote.

**Comment by sevvie** [comment]
Nonetheless, you and I *do* have consensus that clear, defined goals should be our first milestone. ^.~

**Comment by RealRaven2000** [comment]
I have github account just don't know hot to add me to your project: @RealRaven2000 that's me
